src/python_framework_v01/nhd_network_utilities_v01.py

```diff
@@ -380,6 +382,19 @@ def set_supernetwork_data(
             "title_string": "Pocono Test Example",
             "driver_string": "ESRI Shapefile",
             "layer_string": 0,
+            "water_body_parameter_file_type": "Level_Pool",
+            "level_pool_waterbody_parameter_file_path": os.path.join(
+                geo_input_folder, "NWM_2.1_Sample_Datasets", "LAKEPARM_CONUS.nc"
+            ),
+            "level_pool_waterbody_area": "LkArea",  # area of reservoir
+            "level_pool_weir_elevation": "WeirE",
+            "level_pool_waterbody_max_elevation": "LkMxE",
```

Comment:
I would strongly suggest moving all the "level_pool*" entries to a nested dictionary.

Reply:
That makes sense -- these dictionaries are mirrored in the json custom input file format, which ultimately should be the way we provide input to the operational model. A nested dictionary should work with that seamlessly.